common/lib/xmodule/xmodule/video_module/video_xfields.py

```diff
@@ -173,3 +173,12 @@ class VideoFields(object):
         scope=Scope.settings,
         default="",
     )
+    date_last_view_bumper = DateTime(
+        display_name=_("Date of the last view of the bumper"),
+        scope=Scope.preferences,
+    )
+    do_not_show_again_bumper = Boolean(
```

Comment:
And since we can never really update field names, can we at the minimum change `do_not_show_again_bumper` to `bumper_do_not_show_again`? This would kind of namespace the two fields with `bumper_`.